client/components/push-notification/prompt.jsx

```diff
@@ -79,13 +80,25 @@ const PushNotificationPrompt = React.createClass( {
 		return <Notice className="push-notification__notice" text={ noticeText } icon="bell" onDismissClick={ this.props.dismissNotice } />;
 	},
 
+	isUnsupportedChromeVersion: function() {
```

Comment:
This should be put in https://github.com/Automattic/wp-calypso/blob/master/client/state/push-notifications/utils.js#L8, avoiding the need to duplicate the logic and stopping things earlier on.